presto-docs/src/main/sphinx/release/release-0.260.rst

```diff
@@ -2,6 +2,11 @@
 Release 0.260
 =============
 
+.. warning::
+
+   There is a concurrency issue in the metric tracking framework in this release which could cause query failures when Alluxio caching is enabled.
+   optimizer.aggregation-if-to-filter-rewrite-enabled is enabled by default in this release which could cause query failures when the IF branch returns exceptions for rows not meeting the IF condition, e.g, ``SUM(IF(CARDINALITY(array) > 0, array[1]))``.
```

Comment:
double back quote the property name as well.

Reply:
Done.